### PR TITLE
Page template

### PR DESCRIPTION
**Pull request by scriptanium**
The merge status and future merge request where we integrate components into the feed.

### CODE REVIEW

**Comment by scriptanium** [comment]
We need to use themes in colors

**Comment by scriptanium** [comment]
At this state change colors for theme
@CubeLeopard5
We need to choose which article preview because you made some and here is another

File: components/FilterBar.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import { Box } from '@mui/system';
+import { Typography } from '@mui/material';
+import { default as filters } from '../jsonFiles/filterCategory.json';
+
+const Search = styled('div')(({ theme }) => ({
+    position: 'relative',
+    marginTop: '0.625rem',
+    border: '3px solid #7447D5',
```
Comment (ThalusA) [inline components/FilterBar.tsx:16]:
```suggestion
    border: 'solid',
    borderRadius: 3,
    borderColor: 'secondary.main'
```

File: components/FilterBar.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    borderRadius: theme.shape.borderRadius,
```
Comment (ThalusA) [inline components/FilterBar.tsx:17]:
```suggestion
```

File: mocking/articles.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{
+    "articles": [
+        {
+            "id": 0,
+            "title": "Night of the Day of the Dawn of the Son of the Bride of the Return of the Revenge of the Terror of the Attack of the Evil, Mutant, Alien, Flesh Eating, Hellbound, Zombified Living. ",
+            "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce nec fringilla nulla, id efficitur turpis. Donec sed erat euismod, auctor libero in, mollis risus. Pellentesque feugiat sem quis tortor convallis venenatis. Sed et auctor dui. Sed eu consequat libero. Aenean sit amet convallis nunc, eu feugiat ante. Praesent placerat posuere sit amet convallis nunc, eu feugiat ante. Praesent placerat posuere",
+            "image": "camion.jpg",
+            "authors": [
+                "Adina Cazalens",
+                "Alexandre Monnier",
```
Comment (ThalusA) [inline mocking/articles.json:10]:
```suggestion
                "Alexandre Monier",
```

File: mocking/hot_articles.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+    "hotArticles": [
+        {
+            "id": 0,
+            "title": "Night of the Day of the Dawn of the Son of the Bride of the Return of the Revenge of the Terror of the Attack of the Evil, Mutant, Alien, Flesh Eating, Hellbound, Zombified Living. ",
+            "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce nec fringilla nulla, id efficitur turpis. Donec sed erat euismod, auctor libero in, mollis risus. Pellentesque feugiat sem quis tortor convallis venenatis. Sed et auctor dui. Sed eu consequat libero. Aenean sit amet convallis nunc, eu feugiat ante. Praesent placerat posuere sit amet convallis nunc, eu feugiat ante. Praesent placerat posuere",
+            "image": "camion.jpg",
+            "authors": [
+                "Adina Cazalens",
+                "Alexandre Monnier",
```
Comment (ThalusA) [inline mocking/hot_articles.json:10]:
```suggestion
                "Alexandre Monier",
```

**Review by ThalusA — CHANGES_REQUESTED** [review]

**Review by Flavienallix — APPROVED** [review]

**Review by Jerem-b — APPROVED** [review]

**Review by Flavienallix — APPROVED** [review]

**Review by CubeLeopard5 — APPROVED** [review]
c'est bo

**Pull request merged**